more/free-programming-cheatsheets.md

```diff
@@ -238,6 +238,7 @@
 
 * [Beginner's essential jQuery Cheat Sheet](https://websitesetup.org/wp-content/uploads/2017/01/wsu-jquery-cheat-sheet.pdf) (PDF)
 * [jQuery CheatSheet](https://htmlcheatsheet.com/jquery/) (HTML)
+* [The jQuery Mega Cheat Sheet](https://makeawebsitehub.com/wp-content/uploads/2015/09/jquery-mega-cheat-sheet-Printable.pdf) - Jamie Spencer (PDF)
```

Comment:
title is "jQuery the Mega Cheat Sheet"
when you've made changes, please add `(changes made)` to the title of your PR